src/ZODB/tests/testConnection.py

```diff
@@ -521,29 +529,18 @@ def doctest_invalidate(self):
         Transaction ids are 8-byte strings, just like oids; p64() will
         create one from an int.
 
-        >>> cn.invalidate(p64(1), {p1._p_oid: 1})
+        >>> mvcc_storage.invalidate(p64(1), {p1._p_oid: 1})
```

Comment:
It might be nice if the invalidate and invalidateCache methods stayed on Connection (even if not in the interface) for BWC. Or in a `DeprecatedConnectionMethodsMixin` to help keep Connection itself smaller. They could easily simply pass the args to the storage.